idaes/core/ui/flowsheet.py

```diff
@@ -226,6 +228,8 @@ def _identify_arcs(self):
             self._known_endpoints.add(component.source.parent_block())
             self._known_endpoints.add(component.dest.parent_block())
             self._ordered_stream_names.append(component.getname())
+            if self.add_variable_type:
```

Comment:
I don't understand the approach here. Why not put the variable type in its own data structure, e.g. `self._ordered_stream_types`?